sql/mozfun/bits28/examples/extract_week_specific_bits.sql

```diff
@@ -3,6 +3,6 @@ SELECT
   mozfun.bits28.range(days_seen_bits, -13 + 0, 7) AS week_0_bits,
   mozfun.bits28.range(days_seen_bits, -13 + 7, 7) AS week_1_bits
 FROM
-  telemetry.clients_last_seen
+  `mozfun.telemetry.clients_last_seen`
```

Comment:
Looks like things in `mozfun/*/examples` should probably be manually qualified? I suspect this was meant to be 
```suggestion
  `mozdata.telemetry.clients_last_seen`
```
See far above comment on validation for manually written queries

Reply:
added UDF examples to the format skip list